pupil-api/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM node:8
+
+RUN mkdir -p /usr/src/app
+WORKDIR /usr/src/app
+COPY package.json /usr/src/app
+RUN yarn install --ignore-engines
+
+COPY . .
+RUN yarn build
+RUN yarn global add pm2
+
+EXPOSE 3005
```

Comment:
Port 3005 is exposed, but in the docker-compose files it’s referred through port 3003?

Reply:
good catch, ive switched to 3003 so we now have...
- admin:300
- pupil-spa:3002
- pupil-api:3003